src/API/OnboardingTasks.php

```diff
@@ -785,6 +785,7 @@ public function dismiss_task( $request ) {
 				array(
 					'id'             => $id,
 					'is_dismissable' => true,
+					'parent_id'      => 'extended',
```

Comment:
cc @louwie17 - Added the `parent_id` manually here since this is a temporary fix.  Just want to make sure that all dynamic tasks should belong to the `extended` task list- is that right?